addons/mail/static/src/components/channel_member/channel_member.xml

```diff
@@ -2,8 +2,8 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ChannelMember" owl="1">
-        <div class="o_ChannelMember d-flex align-items-center mx-3 my-1" t-attf-class="{{ className }}" t-att-data-partner-local-id="channelMemberView.channelMember.partner.localId" t-ref="root">
-            <div class="o_ChannelMember_avatarContainer position-relative flex-shrink-0 o_cursor_pointer" t-on-click="channelMemberView.onClickMemberAvatar">
+        <div class="o_ChannelMember d-flex align-items-center mx-2 p-2" t-att-class="{'o_cursor_pointer': channelMemberView.hasOpenChat}" t-attf-class="{{ className }}" t-att-data-partner-local-id="channelMemberView.channelMember.partner.localId" t-att-title="channelMemberView.hasOpenChat ? channelMemberView.memberTitleText : ''" t-on-click="channelMemberView.onClickMember" t-ref="root">
```

Comment:
instead of the condition` channelMemberView.hasOpenChat ? channelMemberView.memberTitleText : ''` being in the template, it should be in the computed field itself instead

Reply:
Should we make an OnChange methdod on `hasAuthorOpenChat` which will update all the fields which rely on it. Because there are lot of fields which is relying on hasAuthorOpenChat